gloo-edge/gloo-edge/README.md

```diff
@@ -507,16 +933,46 @@ To test rate limiting, refresh the browser until you see a 429 message.
 
 ![Browser 429 Too Many Requests Interface](images/4.png)
 
-### Web Application Firewall (WAF)
+```
+/opt/google/chrome/chrome $APP_URL/productpage 
+```
+
 
-A web application firewall (WAF) protects web applications by monitoring, filtering and blocking potentially harmful traffic and attacks that can overtake or exploit them.
+### Rate Limiting For Authenticated Users
 
-Gloo Edge Enterprise includes the ability to enable the ModSecurity Web Application Firewall for any incoming and outgoing HTTP connections. 
+Now, let's change the Rate Limit to be only applied for Authenticated Users. But this time, you will apply the changes directly in the Virtual Service. This will affect all routes.
```

Comment:
I think the explanation isn't really clear. What you are implementing is 2 different rate limits. One for authenticated and one for non authenticated users

Reply:
Done